try:
    
    numero = int(input("Introducí un valor numérico: "))
    #print(f"el valor es el {numero}")
    fgg
except NameError as ex:
    print(f"ocurrrió un error")
except ValueError as ex:
    print("Ocurrió un error nuevamente")        

